deploy.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+
+docker build --build-arg VCS_REF=`git rev-parse --short HEAD`   --build-arg BUILD_DATE=`date -u +”%Y-%m-%dT%H:%M:%SZ”` -t scoreucsc/bassa-prod-ui ui >/dev/null
```

Comment:
Can you add some comments to this script. This is a bit hard to understand. 😁😁😁

Reply:
Yupp :)